packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -55,7 +55,6 @@ contract Staker is StakerInterface, Ownable, Testable {
 
     // Pulls tokens from users wallet and stakes them.
     function stake(uint256 amount) public override {
-        require(delegateToStaker[msg.sender] == address(0), "Staker cant be a delegate");
```

Comment:
Yeah, I think this is a good check to remove. Anything we want to _stop people from doing_ we can do in the UI unless it's a security issue or unrecoverable.